pybossa/view/account.py

```diff
@@ -188,6 +191,15 @@ def signin():
         # User already signed in, so redirect to home page
         return redirect_content_type(url_for("home.home"))
 
+def disable_redirect(brand=None):
```

Comment:
You shouldn't need to pass the `brand` here.  You can just fetch it in this function `brand = current_app.config['BRAND']`

Reply:
agree w/ jaret